app/javascript/lib/highlight.js

```diff
@@ -0,0 +1,18 @@
+// If an element to highlight is declared in the document head, add a CSS class
+// to highlight it. Remove the class when the highlight animation completes
+// so it can be triggered on the same element again if needed.
+export function highlightElement () {
+  const metaTag = document.querySelector("meta[name='highlight-element']")
+  if (metaTag) {
+    const elementIDs = metaTag.getAttribute('content').split(',')
+    elementIDs.forEach(elementID => {
+      const element = document.getElementById(elementID)
+      if (element) {
+        element.addEventListener('animationend', function (event) {
```

Comment:
I don't think it's bad to clean up the class once it's done, but another option to consider is [specifying an `animation-fill-mode` of `forwards` in the css](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode) (I think you can pass it as part of `animation:`). With that you could drop this `animationend` event listener, although the highlight class would remain. I'm also not sure if some of that morphing stuff would mess this more css-centric approach up.

Reply:
Interesting. If the class wasn't removed I think it would mean that a future update done via morphing replacement wouldn't trigger the animation. I'll keep this for now to remove the class, but we can re-visit this if it's ever a problem.